<?xml version="1.0" encoding="UTF-8"?>

<launch>

    <node name="myrobot_pick" pkg="myrobot_pick" type="pick_goal" respawn="false" output="screen">
    </node>

</launch>